integration/simulation/simulation_in_mem_test.go

```diff
@@ -24,11 +36,78 @@ func TestInMemoryMonteCarloSimulation(t *testing.T) {
 		SimulationTime:            15 * time.Second,
 		L1EfficiencyThreshold:     0.2,
 		L2EfficiencyThreshold:     0.32,
-		L2ToL1EfficiencyThreshold: 0.34,
+		L2ToL1EfficiencyThreshold: 0.5,
 	}
 
 	params.AvgNetworkLatency = params.AvgBlockDuration / 15
 	params.AvgGossipPeriod = params.AvgBlockDuration / 3
 
 	testSimulation(t, network.NewBasicNetworkOfInMemoryNodes(), params)
 }
+
+// TestMemObscuroRealEthMonteCarloSimulation runs the simulation against a ganache network
+// 1. Install ganache -> npm install ganache --global
```

Comment:
@otherview I guess we need to give this a separate build tag so it isn't run when Ganache isn't up?

Reply:
Defo. We need a build tag that runs ganache and THEN runs this simulation.
Capturing it as a task 👍